docs/UserGuide.md

```diff
@@ -218,19 +220,50 @@ Examples:
 
 ## Features: Managing Person Relationships
 
+### Listing all relationships : `listRelation`
+
+Shows a list of all current relationships in the address book.
+
+Format: `listRelation`
+
 ### Adding a relationship : `addRelation`
 
-Adds a relationship between two people in the address book.
+Adds a roleless relationship between two people in the address book.
 
 Format: `addRelation UUID1 UUID2 RELATIONSHIP_TYPE`
 
-* Adds the relationship between the person with the specified `UUID1` and the person with the specified `UUID2`.
+* Adds the roleless relationship between the person with the specified `UUID1` and the person with the specified `UUID2`.
 * The `UUID1` and `UUID2` refer to the unique identifiers of the persons shown in the displayed person list.
 * The `UUID1` and `UUID2` **must be valid UUIDs**.
 * The `RELATIONSHIP_TYPE` is case-sensitive and matches the relationship type name exactly.
```

Comment:
None of the relationTypes are case sensitive now. Also I think you could add that if the relationType does not exist, it will add it? Relationtypes must be string though

same goes for the rolebased formatting below